src/dune_rules/glob_files.ml

```diff
@@ -0,0 +1,75 @@
+open! Import
+
+type t =
+  { glob : String_with_vars.t
+  ; recursive : bool
+  }
+
+module Path_split = struct
+  (* A source path, or a build path which has been split into its _build/foo
+     prefix, and its remainder. This allows us to generalize over paths in the
```

Comment:
```suggestion
     prefix and its remainder. This allows us to generalize over paths in the
```
Does Dune files stick with the British spelling as we do with Tarides? If so, generalise instead of -ize.

Reply:
Updated to use the British spelling.